src/AutoRest.CSharp.V3/AutoRest/Plugins/CSharpGen.cs

```diff
@@ -71,12 +62,31 @@ public async Task<GeneratedCodeWorkspace> ExecuteAsync(Task<CodeModel> codeModel
                 project.AddGeneratedFile($"{responseHeaderModel.Type.Name}.cs", headerModelCodeWriter.ToString());
             }
 
-            foreach (var client in context.Library.Clients)
+            if (!context.Configuration.AzureArm)
```

Comment:
As I mentioned in the last PR (?), we have 

```
if (Foo)
{
}
if (!Foo)
{
}
```
Can we convert this to an if else?